src/services/tree-sitter/index.ts

```diff
@@ -148,12 +180,20 @@ async function parseFile(
 
 			// Add separator if there's a gap between captures
 			if (lastLine !== -1 && startLine > lastLine + 1) {
-				formattedOutput += "|----\n"
+				if (addLineNumbers) {
+					formattedOutput += "||    ||----\n"
```

Comment:
Might be nice to add some logic around `maxLineNumberWidth` like https://github.com/RooVetGit/Roo-Code/blob/99f64cada15972cea60cf243b8a7704d8331fad7/src/integrations/misc/extract-text.ts#L57-L66